### ISSUE TITLE
Asset ticker and icon not shown.

### ISSUE THREAD

**Issue by Sami32** [tdex-network/tdex-app#602]
It doesn't display my USDt.
I cannot swap my USDt to L-BTC, like i do with other App like SideSwap.
Android 7.0 (Xiaomi)
TDEX 2.0.0 18

![Screenshot_2024-12-11-18-41-46-206_io sevenlabs app](https://github.com/user-attachments/assets/4baf9566-5dd6-4b5a-a81f-bbe37f19a8ce)
![Screenshot_2024-12-11-18-41-37-973_io sevenlabs app](https://github.com/user-attachments/assets/20423805-aaec-4744-972d-533b8ea5acb9)
![Screenshot_2024-12-11-18-54-46-942_io sevenlabs app](https://github.com/user-attachments/assets/f0d3b3e3-fc59-4295-b775-59f72d4c91bd)

**Comment by tiero** [comment]
Please reload your app (kill and reopen), the fact is not showing the right icon and ticker is no bueno. I suspect Block stream registry is failing.

**Comment by Sami32** [comment]
Nope...i probably should have said that uninstalling and reinstalling TDEX didn't removed this issue.

It is just a bug affecting Xiaomi devices or old Android versions?

**Comment by tiero** [comment]
The fact you dont' see the icons and right name of the assets is not a good sign. We have currently no bandwith for support, I would suggest post in the Telegram group of TDEX and Liquid Dev for help (remember this is a reference open source implementation, not a commercial project)

**Comment by Sami32** [comment]
Since i don't have any Telegram or any social account i don't plan to.

At least this issue is now listed here so people having the same problem will be able to know that they are not alone.

Edit:
SideSwap & Green wallet can be used without issue.
